legacy/public/css/playlist_builder.css

```diff
@@ -658,7 +658,7 @@ li.spl_empty {
 .collapsible-header.visible .arrow-icon {
     transform: rotate(90deg);
 }
-.smart-block-advanced {
+.smart-block-advanced, .visual-waveform-editor {
```

Comment:
I picked this since it's the umbrella CSS for the playlist builder with all its tabs and a similar collapsible bar is used in the smart block builder. I don't think we have a special CSS for just the library editor.